source/catalog.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Седона</title>
+  </head>
+  <body class="page__body">
+    <header class="main-header">
+      <nav class="main-nav">
+        <a class="main-header__logo">
+        <img src="/source/img/header-logo.svg" width="185" height="96" alt="Город Седона.">
+        </a>
+        <button class="main-nav__toggle" type="button">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>
+        <div class="main-nav__wrapper">
+          <ul class="main-nav__list site-list">
+            <li class="site-list__item site-list__item--current">
+              <a>
+                Главная
+              </a>
+            </li>
+            <li class="site-list__item">
+              <a href="#">
+                Фото и Видео
+              </a>
+            </li>
+            <li class="site-list__item">
+              <a href="#">
+                Форма отзыва
+              </a>
+            </li>
+            <li class="site-list__item">
+              <a href="#">
+                HTML ACADEMY
+              </a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <main class="media">
+      <header class="media_header hero">
```

Comment:
Здесь и далее опечатка в названии класса элемента. Должно быть 2 нижних подчёркивания:
```suggestion
      <header class="media__header hero">
```

Проверить правильность наименования классов по БЭМ можно [здесь](https://yoksel.github.io/html-tree/).